src/main/java/org/ice4j/message/Message.java

```diff
@@ -1058,7 +1063,7 @@ public static Message decode(byte binMessage[], char offset, char arrayLen)
         int originalOffset = offset;
         arrayLen = (char)Math.min(binMessage.length, arrayLen);
 
-        if(binMessage == null || arrayLen - offset < Message.HEADER_LENGTH)
+        if(arrayLen - offset < Message.HEADER_LENGTH)
```

Comment:
why do we no longer need the null check?

Reply:
Because we reference binMessage.length on the line above

Reply:
😭